site/components/form/Map.tsx

```diff
@@ -0,0 +1,23 @@
+import { CSSProperties, ReactElement } from "react";
+import Map, { ViewState } from "react-map-gl";
```

Comment:
We can update the name of this default import to MapBox to get rid of the lint warning and then change the name of the component from MapBox to Map

Reply:
done